pysmt/smtlib/printers.py

```diff
@@ -247,6 +247,67 @@ def walk_str_to_int(self,formula, **kwargs):
         self.walk(formula.arg(0))
         self.write(")")
 
+    def walk_str_to_re(self,formula, **kwargs):
```

Comment:
All of these should be implemented as a call to walk_binary or walk_nary. See examples above.